### PR TITLE
writable-paths: add /etc/tmpfiles.d for systemd-tmpfiles configs

### PR DESCRIPTION
**Pull request by alexmurray**
Signed-off-by: Alex Murray <alex.murray@canonical.com>

Thanks for helping us make a better ubuntu core!

Before continuing with the PR, you might want to consider also creating a PR for the new core-base repository
at https://github.com/snapcore/core-base, which contains newer core versions (22+), if the PR is also relevant
for newer core versions.

### CODE REVIEW

File: static/etc/system-image/writable-paths [hunk 1]
```diff
@@ -65,3 +65,5 @@
 /etc/machine-id                         auto                    persistent  transition  none
 # make update-motd writable for branding purposes
 /etc/update-motd.d                      auto                    persistent  transition  none
+# for systemd-tmpfiles configurations
+/etc/tmpfiles.d                         auto                    persistent  transition  none
```
Comment (alfonsosanchezbeato) [inline static/etc/system-image/writable-paths:69]: Nitpick: this is complaining about missing newline at EOF

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks

File: static/etc/system-image/writable-paths (hunk 1 above)
Comment (alexmurray) [reply static/etc/system-image/writable-paths]: thanks - I've just fixed this up and force-pushed.

**Comment by alexmurray** [comment]
Closing this as it doesn't appear to be required after all at this stage.

**Pull request closed without merging**